src/EditorFeatures/Test2/Rename/RenameEngineResult.vb

```diff
@@ -115,7 +116,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
 
             Dim renameOptions = RenameOptionSet.From(solution, optionSet)
 
-            If host = TestHost.OutOfProcess_SplitCall Then
+            If host = TestHost.OutOfProcess Then
```

Comment:
this is differnet than before.  i am testing three cases:

1. in process.
2. a single OOP call to get resolved locations
3. an oop call to get locations, and another oop call to resolve them.  

Note; i want to maintain this behavior in tests as it signifies real supported api call patterns that we have.